packages/kbn-eslint-import-resolver-kibana/import_resolver_kibana.js

```diff
@@ -66,9 +68,20 @@ function tryNodeResolver(importRequest, file, config) {
   );
 }
 
+function isImportedFromRoot(importRequest) {
```

Comment:
As I understand `packages/kbn-eslint-plugin-eslint/rules/no_restricted_paths.js` should integrate this resolution logic as well. Can we extract it in a common folder to share? Can be done in a separate PR.